tree.go

```diff
@@ -265,15 +265,14 @@ func (t *Tree) addseg(segments []string, route interface{}, wildcards []string,
 			}
 			t.wildcard.addseg(segments[1:], route, append(wildcards, params...), reg+regexpStr)
 		} else {
-			var ok bool
 			var subTree *Tree
-			for _, subTree = range t.fixrouters {
-				if t.prefix == seg {
-					ok = true
+			for _, sub := range t.fixrouters {
```

Comment:
the code is more readable,but i can not find any clue to #1595. i think there's no difference,or there's is what i  have ignored?